janus.c

```diff
@@ -4996,6 +4996,11 @@ gint main(int argc, char *argv[])
 	SSL_library_init();
 	SSL_load_error_strings();
 	OpenSSL_add_all_algorithms();
+	/* check if random pool looks ok (this does not give any guarantees for later, though) */
```

Comment:
Just a nit on code syle: comments should start with a capitized letter.

Reply:
done